app/pages/sections/order-items/catalogue-solutions/select/price/ui-tests/general.ui.test.js

```diff
@@ -89,6 +110,9 @@ const pageSetup = async (setup = defaultPageSetup) => {
   if (setup.postRoute) {
     await setState(ClientFunction)(sessionKeys.solutionPrices, solutionPricesInSession);
   }
+  if (setup.onePrice) {
+    await setState(ClientFunction)(sessionKeys.selectedItemId, selectedItemIdInSession);
+  }
```

Comment:
I don't think we need this additional if clause. Should just be able to use the getRoute setup for the ui test